ScoutSuite/frontend/src/partials/gcp/computeengine.projects.id.zones.id.instances.id/index.js

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash/get';
+import { Link } from 'react-router-dom';
+
+import { Partial, PartialValue } from '../../../components/Partial';
+import {
+  partialDataShape,
+  formatDate,
+  convertBoolToEnable,
+} from '../../../utils/Partials';
+import { TabsMenu, TabPane } from '../../../components/Tabs';
+import GetResourceName from '../../../components/GetResourceName';
+import { valueOrNone } from '../../../utils/Partials/index';
+
+const renderTags = tags => {
+  if (!tags || tags.length === 0) return <span>None</span>;
+  return (
+    <div className="partial-value detailed-value">
+      <span className="label">Tags</span>
+      <ul>
+        {Object.entries(tags).map(([key, value]) => (
+          <li key={key}>
+            {key}
+            <ul>
+              {value &&
+                Array.isArray(value) &&
+                value.map((v, i) => <li key={i}>{v}</li>)}
+              {value && !Array.isArray(value) && <li>{value}</li>}
+              {!value || (value.length === 0 && <li>None</li>)}
+            </ul>
+          </li>
+        ))}
+      </ul>
+    </div>
+  );
+};
+
+const renderNetworkInterfaces = network_interfaces => {
+  if (!network_interfaces) return <span>None</span>;
+
+  return (
+    <ul>
+      {network_interfaces.map((ni, i) => (
+        <li key={i}>
+          {ni.name}
+          <ul>
+            <li>{ni.networkIP}</li>
+            <li>
+              Network:{' '}
+              <Link
+                to={`/services/computeengine/resources/networks/${ni.network_id}`}
+              >
+                <GetResourceName
+                  service="computeengine"
+                  resource="networks"
+                  id={ni.network_id}
+                />
+              </Link>
+            </li>
+            <li>
+              Subnetwork:{' '}
+              <Link
+                to={`/services/computeengine/resources/subnetworks/${ni.subnetwork_id}`}
+              >
+                <GetResourceName
+                  service="computeengine"
+                  resource="subnetworks"
+                  id={ni.subnetwork_id}
+                />
+              </Link>
+            </li>
+          </ul>
+        </li>
+      ))}
+    </ul>
+  );
+};
+
+const renderAccessScope = access_scopes => {
+  if (!access_scopes) return <span>None</span>;
+  return (
+    <ul>
+      {access_scopes.map((item, i) => (
+        <li key={i}>{item}</li>
+      ))}
+    </ul>
+  );
+};
+
+const renderDisks = (disks) => {
+  if (!disks) return <span>None</span>;
+
+  return <ul>
+    {Object.values(disks).map((disk, i) => <li key={i}>
+      {disk.source_device_name}
+      <ul>
+        <li>Bootable: {disk.bootable}</li> 
+        <li>Type: {disk.type}</li>
+        <li>Mode: {disk.mode}</li>
+        {disk.latest_snapshot ?
```

Comment:
you could use the ternary expression only for the value rather than the whole element so you dont have to repeat the label in both.

Reply:
Good point!

Reply:
Fixed